src/main/java/frc/robot/subsystems/Limelight.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.apriltag.AprilTagFields;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj.DriverStation;
+import frc.robot.constants.VisionConstants;
+import java.io.IOException;
+import java.util.Optional;
+import org.photonvision.EstimatedRobotPose;
+import org.photonvision.PhotonCamera;
+import org.photonvision.PhotonPoseEstimator;
+import org.photonvision.PhotonPoseEstimator.PoseStrategy;
+
+public class Limelight {
+    private final PhotonCamera mPhotonCamera;
+    private PhotonPoseEstimator mPhotonPoseEstimator;
```

Comment:
Make final.

Reply:
The PhotonPoseEstimator throws an exception if the field fails to load which makes us need to wrap it with a try-catch so it cannot be final.